### ISSUE TITLE
Very great work! But I noticed what seems to be an inconsistency between the paper and the code.

### ISSUE THREAD

**Issue by Baobaon** [ZhaoYuTJPU/MSSGCL#1]
The implementation of the code to compute the loss for global and local samples is as follows:
![image](https://github.com/user-attachments/assets/ac6adf58-c2a6-42e1-88c2-2657a1107e63)
where loss = pos_sim / (sim_matrix.sum(dim=1) - pos_sim) does not match the paper equation：
![image](https://github.com/user-attachments/assets/37c9386d-06df-4531-83e2-2d8c3a676361)
I'd like your help!
